chapter_chai/frontend/src/map-page/MapPage.js

```diff
@@ -106,7 +106,7 @@ function MapPage() {
             <LoadScript
 
                 loadingElement={<div>Loading...</div>}
-                googleMapsApiKey={process.env.REACT_APP_GOOGLE_API_KEY}
+                googleMapsApiKey="AIzaSyAQzSw091TkcMWpTUrwP54WJH2jN-6pzKo"
```

Comment:
I think you left the google maps api key here by accident. Can you remove it? Thanks!!

Reply:
Done!